pkg/grpc/config/config.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
```

Comment:
this import would read ` pkg/grpc/config`, which doesn't look right considering that it's specific to storage. Any reason not to have it under `plugin/storage/grpc`?

Reply:
I had just left it as it was when Olivier wrote it, now moved.